model/Image.php

```diff
@@ -1051,3 +1051,14 @@ public function write($showDebug = false, $forceInsert = false, $forceWrite = fa
 		throw new Exception("{$this->ClassName} can not be written back to the database.");
 	}
 }
+
```

Comment:
> Isn't there any kind of helper file where global methods can be defined?

Global text conversion methods are in `Convert` :D

We tend to avoid creating global methods where possible. In this case, it's fine to scope it to a class, since these aren't really standard php methods (even if they exist on other languages).

you can call it whatever you want as long as it's documented and makes sense :P